publiccode.yml

```
@@ -0,0 +1,37 @@
+publiccodeYmlVersion: "1.0"
+name: DSFR
+url: https://github.com/GouvernementFR/dsfr
+landingURL: https://www.systeme-de-design.gouv.fr/
+creationDate: 2021-04-08
+latestRelease:
+  date: 2025-03-26
+  version: "1.13.1"
+logo: ""
+usedBy: []
+fundedBy:
+  - name: Service d'Information du Gouvernement
+    url: https://www.info.gouv.fr/organisation/service-d-information-du-gouvernement-sig
+roadmap: "https://www.systeme-de-design.gouv.fr/a-propos/roadmap"
+softwareType: "standalone/web"
+description:
+  fr:
+    shortDescription: Système de Design de l'État - DSFR
+    documentation: https://www.systeme-de-design.gouv.fr/
+legal:
+  license: MIT
+  repoOwner: Service d'Information du Gouvernement
+maintenance:
+  type: internal
+  contacts:
+    - name: "Maxime Beaugrand"
+      email: "maxime.beaugrand@pm.gouv.fr"
+metadataFiles:
+  readme: README.md
+  license: LICENSE.md
+  contributing: CONTRIBUTING.md
+  changelog: CHANGELOG.md
+  codeOfConduct: null
+  governance: null
+lastUpdated: 2025-03-26
```

```suggestion
lastUpdated: 2025-03-26
screenshots: |-
  https://www.systeme-de-design.gouv.fr/uploads/banner_img_aaa47076b8.png
genericName: dsfr
```
lastUpdated: 2025-03-26
```